WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReaders/CardReaderSettingsConnectView.swift

```diff
@@ -0,0 +1,197 @@
+import UIKit
+
+final class CardReaderSettingsConnectView: NSObject {
+
+    private var rows = [Row]()
+
+    var onPressedConnect: (() -> ())?
+
+    override init() {
+        super.init()
+        rows = [.connectHeader, .connectImage, .connectHelp1, .connectHelp2, .connectHelp3, .connectButton, .connectLearnMore]
+    }
+
+    public func rowTypes() -> [UITableViewCell.Type] {
+        return [
+            ButtonTableViewCell.self,
+            ImageTableViewCell.self,
+            LearnMoreTableViewCell.self,
+            NumberedListItemTableViewCell.self,
+            TitleTableViewCell.self
+        ]
+    }
+
+    private func configure(_ cell: UITableViewCell, for row: Row, at indexPath: IndexPath) {
+        switch cell {
+        case let cell as TitleTableViewCell where row == .connectHeader:
+            configureHeader(cell: cell)
+        case let cell as ImageTableViewCell where row == .connectImage:
+            configureImage(cell: cell)
+        case let cell as NumberedListItemTableViewCell where row == .connectHelp1:
+            configureHelp1(cell: cell)
+        case let cell as NumberedListItemTableViewCell where row == .connectHelp2:
+            configureHelp2(cell: cell)
+        case let cell as NumberedListItemTableViewCell where row == .connectHelp3:
+            configureHelp3(cell: cell)
+        case let cell as ButtonTableViewCell where row == .connectButton:
+            configureButton(cell: cell)
+        case let cell as LearnMoreTableViewCell where row == .connectLearnMore:
+            configureLearnMore(cell: cell)
+        default:
+            fatalError()
+        }
+    }
+
+    private func configureHeader(cell: TitleTableViewCell) {
+        cell.titleLabel?.text = NSLocalizedString(
+            "Connect your card reader",
+            comment: "Settings > Manage Card Reader > Prompt user to connect their first reader"
+        )
+        cell.hideSeparator()
+        cell.selectionStyle = .none
+    }
+
+    private func configureImage(cell: ImageTableViewCell) {
+        cell.detailImageView?.image = UIImage(named: "card-reader-connect")
+        cell.hideSeparator()
+        cell.selectionStyle = .none
+    }
+
+    private func configureHelp1(cell: NumberedListItemTableViewCell) {
```

Comment:
Would it make sense to name these methods in a way that indicate what each help button does? Maybe primaryHelp, secondaryHelp...

Reply:
Yep! Fixed in https://github.com/woocommerce/woocommerce-ios/pull/3699/commits/6d9903a53c2aa579399e2de43657d9c805077611